EIPS/eip-7702.md

```diff
@@ -73,7 +73,8 @@ At the start of executing the transaction, after incrementing the sender's nonce
 5. Verify the code of `authority` is either empty or already delegated.
 6. Verify the nonce of `authority` is equal to `nonce`. In case `authority` does not exist in the trie, verify that `nonce` is equal to `0`.
 7. Add `PER_EMPTY_ACCOUNT_COST - PER_AUTH_BASE_COST` gas to the global refund counter if `authority` exists in the trie.
-8. Set the code of `authority` to be `0xef0100 || address`. This is a delegation designation.
+8. Set the code of `authority` to be `0xef0100 || address`. This is a delegation designation. 
+    * As a special case, if `address` is `0x0000000000000000000000000000000000000000` do not write the designation. Clear the accounts code and reset the account's code hash to `0x0000000000000000000000000000000000000000000000000000000000000000`.
```

Comment:
Why is the account code hash set to these zeros? It should be set to `keccak256("")`?

Reply:
Ah you're right. Let me update it.